src/toil/job.py

```diff
@@ -362,6 +363,9 @@ def startToil(job, options):
     failed jobs
             """
             setLoggingFromOptions(options)
+            # Set up real-time log message reception, and put the details in the
+            # environment to be sent out to workers.
+            RealtimeLogger.start_master()
```

Comment:
This should be made optional via a option. The port should also be configurable (if it isn't already).